crates/turborepo-updater/src/lib.rs

```diff
@@ -113,8 +115,17 @@ pub fn display_update_check(
 
     if let Ok(Some(version)) = version {
         let latest_version = version.to_string();
-        // TODO: make this package manager aware
-        let update_cmd = style("npx @turbo/codemod@latest update").cyan().bold();
+
+        let update_cmd = match package_manager {
```

Comment:
Not blocking, but this will have to be undone down the road. Adding a `package_manager.exec_command()` would be preferred to avoid doing an explicit iteration over all supported package managers.

e.g.
```
fn exec_command(&self) -> &str {
    match self {
        PackageManager::Npm => "npx",
        PackageManager::Yarn | PackageManager::Berry => "yarn dlx",
        PackageManager::Pnpm | PackageManager::Pnpm6 | PackageManager::Pnpm9 => "pnpm dlx"
        PackageManager::Bun =>"bunx",
    }
}
```